docker-compose.yml

```diff
@@ -14,9 +14,11 @@ services:
       QPC_SERVER_USERNAME: admin
       NETWORK_CONNECT_JOB_TIMEOUT: 6
       NETWORK_INSPECT_JOB_TIMEOUT: 600
+      QPC_DEBUGPY: 1
```

Comment:
Should this default to `0`, assuming `0` means off?

Reply:
yeah, zero means off!

I set to `1` considering docker-compose is meant for developers, not end-users :thinking: 

But if you prefer I can change anyway. its easy to override docker-compose locally without actually editing the file :thinking:

Reply:
No, that makes sense, let's keep it `1`. Thanks for the clarification